packages/cursorless-engine/src/core/Cheatsheet.ts

```diff
@@ -41,6 +50,10 @@ export async function showCheatsheet({
 
   const root = parse(cheatsheetContent);
 
+  // Add usage stats here option #2
+  // Like ... document.cheatsheetUsageStats
+
+  // Inject data into the cheatsheet
```

Comment:
Yeah this is prob where I'd do it. Just add them to the payload that gets injected below